source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
This could bloat the pr, so maybe it's not a good idea to make here (we could make another issue).